src/Health/Router.ts

```diff
@@ -67,4 +69,16 @@ export class Router {
       logger.error({ error }, 'Failed to update ipfsHashFailures on health')
     }
   }
+
+  blockDownloadedConsumer = async (blockDownloaded: BlockDownloaded): Promise<void> => {
+    const logger = this.logger.child({ method: 'blockDownloadedConsumer' })
+
+    logger.trace({ blockDownloaded }, 'Block downloaded, removing associated transactions')
```

Comment:
Can you make this one a `debug` instead of `trace`?

Reply:
yep